registry/github_registry.go

```diff
@@ -38,6 +39,7 @@ type githubRegistry struct {
 	format         common.RegistryFormat
 	credentialName string
 	service        GithubRepositoryService
+	httpClient     *http.Client
```

Comment:
I didn't see an easy way when I was writing this. Unless you feel very strongly about this, I'd like to leave it as is, or if you can point how to get it from the service :)